src/client.rs

```diff
@@ -691,3 +833,48 @@ fn test_dai_block_response() {
         assert!(val.number > 10u32.into());
     });
 }
+
+/// Testing all function that involve a syncing node check
+#[test]
+fn test_syncing_check_functions() {
```

Comment:
since this test isn't tagged with ignore it will run all the time. Since there are no unwraps it will never fail. It should have the ignore tag so that it doesn't run all the time or unwrap the responses so it at least provides some level of sanity checking.